src/odemis/acq/move.py

```
@@ -125,7 +125,7 @@ def __init__(self, microscope):
         pass
 
     @abstractmethod
-    def getCurrentPostureLabel(self, pos: Dict[str, float] = None) -> int:
+    def get_current_posture_label(self, pos: Dict[str, float] = None) -> int:
```

Ruff (0.15.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)